### ISSUE TITLE
Show subway lines at lower zoom levels

### ISSUE THREAD

**Issue by njohnston** [osmandapp/Osmand#3905]
I recently noticed that subway lines are only shown from zoom 18 onwards. (OsmAnd 2.6.5.)

This leads to noticeable gaps where the lines should be, particularly as the stations (and platforms) are shown. Particularly at higher zoom levels, the stations are like "islands".

Examples from [north London](http://www.openstreetmap.org/#map=18/51.59525/-0.24940) (I've added red lines where the subway lines/tracks should be):

Zoom 15:

![osmand_subway_line_missing_z15_small](https://user-images.githubusercontent.com/638458/27003613-fb10db4a-4df1-11e7-8b49-bdb0a3bd99f6.png)

Zoom 16:

![osmand_subway_line_missing_z16_small](https://user-images.githubusercontent.com/638458/27003619-0e9cbeae-4df2-11e7-97e6-b006707efd2f.png)

Zoom 17:

![osmand_subway_line_missing_z17_small](https://user-images.githubusercontent.com/638458/27003624-1f50f5e4-4df2-11e7-9a78-bd171f1b5250.png)

I think lines should be shown from zoom 15 at least, possibly 14, because stations (and symbols) are shown at these levels.

I tried to copy the relevant part from default.render.xml into my custom rendering file but got exceptions. What's the minimum I need to put into my custom rendering file to change subway line rendering there? Thanks.

**Comment by xmd5a2** [comment]
Subway lines are rendered from lower zooms when "Subway routes" option is on. These lines were removed from zoom 12-17 in default mode because most user don't need them on these zooms. This causes the map to be cluttered with unnecessary information.

**Comment by njohnston** [comment]
@xmd5a2: I agree that the lines shouldn't be shown at zoom 12 and 13, but other zoom levels, the lines don't clutter the map: they actually complete it. Without the lines, there are noticeable gaps in the map (see my screenshots) and IMHO it's strange to clearly show stations including platforms but not tracks.

Could you tell me the minimum I need to put in my custom rendering file to achieve this? Thanks :)
